src/config.rs

```diff
@@ -500,6 +502,10 @@ macro_rules! build_cf_opt {
         if $opt.enable_doubly_skiplist {
             cf_opts.set_doubly_skiplist();
         }
+        cf_opts.set_compact_on_deletion(
```

Comment:
Why not use deletion_ratio? Especially for small cf like LOCK.

Reply:
what is `deletion_ratio`?

Reply:
Oh, I was looking at upstream rocksdb: https://github.com/facebook/rocksdb/pull/6806. Maybe pick it?

Reply:
The window limit is 10000, I think our LOCK is large enough to fit it